src/ai/backend/manager/api/vfolder.py

```diff
@@ -1257,9 +1257,18 @@ async def mkdir(request: web.Request, params: Any, row: VFolderRow) -> web.Respo
             "parents": params["parents"],
             "exist_ok": params["exist_ok"],
         },
-    ):
-        pass
-    return web.Response(status=201)
+    ) as (_, storage_resp):
+        storage_reply = await storage_resp.json()
+        resp_code = storage_resp.status
+        resp = {
+            "failure_tasks": storage_reply["failure_tasks"],
+            "success_tasks": storage_reply["success_tasks"],
+        }
```

Comment:
The storage proxy implementation does NOT ALWAYS return these fields in the reply.
For example, it may return "msg" and "errno" when all given paths failed to be created.
We should define the consistent form of the API reply.

Reply:
I revised and refactored this part by introducing reusable `common.types.{ItemResult,ResultSet}` definitions.